src/test/resources/config/test_config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config>
+    <jdbc>
+        <url>mysql://testbox:3306</url>
```

Comment:
Hmm what's this?

Reply:
Thanks, Peter.

This is a private test-resource copy of the config that's used exclusively by the ConifgurationBuilderTest unit test (here)[https://github.com/XIVStats/XIVStats-Gatherer-Java/pull/30/files#diff-32f50191e9fba98cccc42dfdd2795d2dR41], to verify the behaviour of the XML configuration code. It doesn't get used in any actual DB configuration for the full-stack integration tests beyond this.

I've placed it under the `test/resources` folder so that it's only available during testing builds and never during final JAR generation, and it uses a separate name so that it won't get picked-up by any of the other config-loading calls.